docs/pages/checkbox.js

```diff
@@ -757,6 +757,88 @@ function CheckboxExample() {
     </Fieldset>
   );
 }
+`}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Label visibility"
+          description={`In some cases, the label for a Checkbox is represented in a different way visually, as demonstrated below. In these instances, you can set labelDisplay="hidden" to ensure Checkbox is properly labeled for screen readers while using a different element to represent the label visually.`}
```

Comment:
nit: wrap in back-ticks  \`labelDisplay="hidden"\`

Reply:
to get the styling

Reply:
Done